cpp/docs/DEVELOPER_GUIDE.md

```diff
@@ -530,6 +530,30 @@ Note:  `std::tuple`  _could_  be used if not for the fact that Cython does not s
 only two objects of different types. Multiple objects of the same type may be returned via a 
 `std::vector<T>`.
 
+Alternatively, with C++17, [structured binding](https://en.cppreference.com/w/cpp/language/structured_binding) 
```

Comment:
Do we want to call out when cudf started to support C++17? Reading this makes me wonder if it can be used or if I need to check the compiler version.

Reply:
I was wondering whether that needed calling out. Thanks for sharing that concern. Fixed.